### PR TITLE
local run enhancements

### PR DESCRIPTION
**Pull request by icemachined**
### What's done:
* additional properties and build tasks were introduced to copy save-agent and save-cli from local wsl folders
* fix agent stop check
* readme updated

### CODE REVIEW

File: save-orchestrator/build.gradle.kts [hunk 1]
```diff
@@ -23,20 +23,29 @@ tasks.withType<KotlinCompile> {
...
-@Suppress("GENERIC_VARIABLE_WRONG_DECLARATION")
+val downloadSaveCliDistribTaskProvider: TaskProvider<Download> = tasks.register<Download>("downloadSaveCliDistrib") {
+    src(KotlinClosure0(function = { findProperty("saveCliDistribFilepath") ?: "file:\\\\" }))
+    File("$buildDir/libs/").mkdirs()
+    dest("$buildDir/libs")
+    overwrite(false)
+}
+
+downloadSaveCliDistribTaskProvider.configure {
+    enabled = findProperty("saveCliDistribFilepath") != null
```
Comment (petertrr) [inline save-orchestrator/build.gradle.kts:34]: I think `enabled` can be set inside `register`?

File: save-deploy/README.md [hunk 2]
```diff
@@ -72,6 +72,18 @@ Usually not the whole stack is required for development. Application logic is pe
 * To make things easier, add line `save.profile=dev` to `gradle.properties`. This will make project version `SNAPSHOT` instead of timestamp-based suffix and allow caching of gradle tasks.
 * Run `./gradlew deployLocal -Psave.profile=dev` to start the database and run three microservices (backend, preprocessor and orchestrator) with Docker Compose.
 
+## Local debugging
+You can run backend, orchestrator, preprocessor and frontend locally in IDE in debug mode.<br/>
+If you run on windows, dependency `save-agent` is omitted because of problems with linking in cross-compilation.<br/>
+To run on windows and you need to compile save-agent on wsl and put saveCliDistribFilepath to %USERPROFILE%\.gradle\gradle.properties <br/>
+For example: `saveCliDistribFilepath=\\home\\username\\projects\\save-cloud\\save-agent\\build\\libs\\save-agent-0.3.0-alpha.0.48+1c1fd41-distribution.jar` <br/>
```
Comment (petertrr) [inline save-deploy/README.md:79]: Shouldn't this path start with `\wsl$` too?

Comment (icemachined) [reply save-deploy/README.md]: fixed

File: save-orchestrator/build.gradle.kts (hunk 1 above)
Comment (icemachined) [reply save-orchestrator/build.gradle.kts]: fixed

**Review by petertrr — APPROVED** [review]

**Pull request merged**